theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/KeyframeDot.tsx

```diff
@@ -135,18 +149,56 @@ function useKeyframeContextMenu(
     },
   })
 }
+/*
+I think <DeterminePropEditor> and its prop-specific components need the following changes to make them suitable for inline editing:
+ 1. Perhaps a react prop/context that says whether the prop editor is used in the DetailsPanel or in an InlineKeyframeEditorPopover. And in the case of the latter:
+ 2. Use a simpler replacement for useEditingToolsForPrimitiveProp() that works per-keyframe.
+ 3. Not render <NextPrevKeyframeCursors />
+ 4. Also, the label of the top-level prop (the prop that the track the keyframe was shown in belongs to) would be hidden.
+*/
```

Comment:
Refactor comment to summarize how these ideas were incorporated? Maybe move this to `KeyframeDeterminePropEditor`?

Reply:
ok